src/app.ts

```diff
@@ -0,0 +1,22 @@
+import { NotionAdapter } from "./notion";
+
+export interface AppConstructor {
+    notion: {
+        api_key: string,
+        database_id: string
+    },
+    blogTokens: {
+        dev: string,
+        hashnode: string
+    }
+}
+
+export class App {
+    private readonly notionAdapter: NotionAdapter;
+    constructor(config: AppConstructor){
+        this.notionAdapter = NotionAdapter.instantiate({
+            api_key: config.notion.api_key,
+            database_id: config.notion.database_id
+        });
+    }
+}
```

Comment:
We have to figure out the interface of the library

Reply:
This is still not resolved I believe, correct??

Reply:
No I am still thinking of a better UX for the library open to any suggestions.